python/cudf/cudf/core/series.py

```diff
@@ -3636,6 +3636,12 @@ def label_encoding(self, cats, dtype=None, na_sentinel=-1):
         4   -1
         dtype: int8
         """
+        
+        warnings.warn(
+            "Series.label_encoding is deprecated and will be removed "
+            "in the future.",
```

Comment:
Is there an alternative approach we want to suggest here?

Reply:
According to issue description, the core functionality is already present in the form of `LabelEncoder` in `cuML`. Is there something else that is planned? I assumed the end goal is to remove the functionality altogether.

Reply:
That's perfect! Would suggest we add that in the warning message: e.g., `"Series.label_encoding is deprecated and will be removed in the future. Consider using cuML's LabelEncoder instead."`.

Reply:
Yeah that sounds better, thanks (and updated)!